Add buttons to traverse through the Journal forwards and backwards

bitmuster opened the pull request:
Hi, here is a small enhancement to add little buttons to traverse through the Journal forwards and backwards easily.
I hope you like it:
![image](https://user-images.githubusercontent.com/14337589/111043328-44af9e00-8442-11eb-96d0-e742ed0638c7.png)

BTW I see some failing tests, but they don't seem to be related...

bitmuster commented:
Sorry, one commit slipped in while rebasing from master to develop. It's reverted now

jaap-karssenberg commented:
Hi Michael, I like the idea, 2 suggestions for improvement

1) instead of using buttons with text "<" and ">" it looks better with buttons containing an icon, try `Gtk.Button.new_from_icon_name()` with "pan-start-symbolic" and "pan-end-symbolic" 

2) when the journal is configured to use weeks instead of days, it would be nice if the buttons skipped a whole week rather than a day at a time - you can check the "granularity" property which can be day/week/month/year and use that as a timedelta

Regards,

Jaap

abelikt commented:
Hello Jaap, thanks a lot for the feedback and the suggestions. I will come up with a solution soon

bitmuster commented:
Hello Jaap, I implemented your proposals and updated the pull request. Please have a look if it is good now.
In case that one selects years or months as granularity, we would use days again. There are specific buttons for that granularity, so I'm not sure if we should fall back to days here.
